src/main/java/com/atlassian/bitbucket/jenkins/internal/client/BitbucketClientFactoryImpl.java

```diff
@@ -219,81 +208,12 @@ public BitbucketWebhookSupportedEventsClient getWebhookCapabilities() {
      *         this method will not work
      * @param <T> type to return
      * @return a deserialized object of type T
-     * @throws AuthorizationException if the credentials did not allow access to the given url
-     * @throws NoContentException if a body was expected but the server did not respond with a body
-     * @throws ConnectionFailureException if the server did not respond
-     * @throws NotFoundException if the requested url does not exist
-     * @throws BadRequestException if the request was malformed and thus rejected by the server
-     * @throws ServerErrorException if the server failed to process the request
-     * @throws BitbucketClientException for all errors not already captured
      * @see #makeGetRequest(HttpUrl, TypeReference)
      */
-    <T> BitbucketResponse<T> makeGetRequest(@Nonnull HttpUrl url, @Nonnull Class<T> returnType) {
+    private <T> BitbucketResponse<T> makeGetRequest(@Nonnull HttpUrl url, @Nonnull Class<T> returnType) {
```

Comment:
We have `@Nonnull` by default so we don't need these annotations. Instead, we have to explicitly annotate `@Nullable`s